atlasdb-dbkvs/src/main/java/com/palantir/atlasdb/keyvalue/dbkvs/impl/DbKvs.java

```diff
@@ -388,6 +388,17 @@ public int compare(Entry<Cell, Long> entry1, Entry<Cell, Long> entry2) {
     });
 
     @Override
+    public void deleteRange(TableReference tableRef, RangeRequest range) {
+        runWriteForceAutocommit(tableRef, new Function<DbWriteTable, Void>() {
```

Comment:
On a slightly unrelated note, the work @gbonik is doing to separate the connection pools for our large internal product should allow us to get rid of this autocommit checking logic!